docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  bot:
+    build: .
+    volumes:
+      - .:/bot
```

Comment:
No you mount over /bot, so the file ADDed by the docker file are hidden.

Reply:
Interestingly, if I run `docker-compose run bot touch aloha.txt` I can see that file on my local machine.

Reply:
So even more interestingly, this seems to be an Atom editor issue. From command line I can see the file, but Atom is not able to see it. Thanks for your time!